test/ListUtils.java

```diff
@@ -0,0 +1,32 @@
+package test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class ListUtils {
+
+    private ListUtils() {
+    }
+
+    public static <T> void swapElemWithImmutable(List<T> source, List<T> origin, int index) {
+
+        List<T> unmodList = Collections.unmodifiableList(source);
+
+        origin.set(index, unmodList.get(index));
+
+        for(T list : origin ) {
+            System.out.println(list);
+        }
+
+    }
+
+    public static <T> void swapElem(List<T> source, List<T> origin, int index) {
```

Comment:
>Create utility method, which takes two lists and index and switches elements at given position;

Switch two elements means to take element N from collection A and set it to collection B and take element N from collection B and set it to A.
E.g.
before: A [1 2 3 4 5], B [6 7 8 9 0]
index: 2
after: A [1 2 **8** 4 5], B [6 7 **3** 9 0]